.babelrc

```diff
@@ -1,10 +1,11 @@
 {
   "presets": [
-    "es2015",
-    "react"
+    "env",
```

Comment:
Don't you typically have to pass some configuration options to `env`?

Reply:
Nope. Works out of the box